src/modules/git_commit.rs

```diff
@@ -0,0 +1,52 @@
+use super::{Context, Module, RootModuleConfig};
+use git2::Repository;
+
+use crate::configs::git_commit::GitCommitConfig;
+
+pub fn id_to_hex_abbrev(bytes: &[u8], len: usize) -> String {
+    bytes
+        .iter()
+        .map(|b| format!("{:02x}", b))
+        .take(len)
+        .collect::<Vec<String>>()
+        .join("")
+}
+
+/// Creates a module with the Git commit in the current directory
+///
+/// Will display the commit hash if the current directory is a git repo
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let mut module = context.new_module("git_commit");
+    let config = GitCommitConfig::try_load(module.config);
```

Comment:
Because we don't yet have the logic to disable modules by default, the following condition has to be manually added for now:
https://github.com/starship/starship/blob/master/src/modules/time.rs#L14-L16